spec/features/admin/adjustments_spec.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-require "system_helper"
+require "spec_helper"
```

Comment:
This commit is what we call a "detour". You first did one change and then you undo it later. It makes reviewing slightly more confusing. Maybe you want to keep this in a WIP branch because you expect to change all feature specs at some point but for the purpose of this pull request, it would be nicer to squash these changes into the first commit.